catalogue/webapp/utils/dates.ts

```diff
@@ -62,7 +62,10 @@ export function findNextRegularOpenDay(
   }
   const isClosed = regularClosedDays.includes(date.day() as DayNumber);
   if (isClosed) {
-    return findNextRegularOpenDay(date.add(1, 'day'), regularClosedDays);
+    // If a request is made on a day the library is closed, the next available
+    // pickup day should be the day _after_ the library reopens so library
+    // staff aren't overwhelmed on e.g. Monday mornings
+    return findNextPickUpDay(date.add(2, 'day'), regularClosedDays);
```

Comment:
I'm already not sure about this. If someone makes a request on a Saturday, I think this will result in them being told Monday is the next date.